website/content/v1.0/kubernetes-guides/configuration/ceph-with-rook.md

```diff
@@ -95,8 +95,8 @@ ceph-filesystem        rook-ceph.cephfs.csi.ceph.com   Delete          Immediate
 
 ## Talos Linux Considerations
 
-It is important to note that a Rook Ceph cluster saves cluster information directly onto the node (be default `dataDirHostPath` is set to `/var/lib/rook`) which under Talos Linux is **ephemeral**.
-This makes cluster management a little bit more involved, as any time a Talos Linux node is reconfigured or upgraded, the ephemeral partition is wiped.
+It is important to note that a Rook Ceph cluster saves cluster information directly onto the node (be default `dataDirHostPath` is set to `/var/lib/rook`).
+This makes cluster management a little bit more involved, as any time a Talos Linux node is reconfigured or upgraded, the partition that stores the `/var` [file system]({{< relref "../../learn-more/architecture#the-file-system" >}}) is wiped, but the `--preserve` option of [`talosctl upgrade`]({{< relref "../../reference/cli#talosctl-upgrade" >}}) can be used to avoid this.
```

Comment:
I think we should mention that if there is a odd number of OSD, the preserve flag could be avoided and wait for rook to mark oSD back as healthy after upgrade or manually mark the OSD's as paused (I'm not sure what;s the right ceph wording is here)

Reply:
Number of OSDs doesn't really matter as much as capacity & pool replica count.

In fact, Rook is smart enough to rebuild the mon data stored in the `/var` when the node comes back up. In the guide itself I never use `--preserve` when doing an upgrade since Ceph marks the OSDs as 'down' and not 'out', so when they come back 'up' only the changes have to be synced back for the OSD to be brought back 'in' and the cluster to be 'healthy' again.

Using `--preserve`, like with a non-HA cluster, is only really needed if we only have one `mon`.

Reply:
Ahhh, you mean if there are multiple `mon`s, yeah, that makes more sense! I'll add it.

Reply:
I think by default it always does three mons, so Rook/Ceph won't even deploy with less than 3 worker (schedulable) nodes

Reply:
For sure. It's a balance between what is a consideration for Rook/Ceph or Talos Linux.